modules/custom/openy_campaign/src/Form/ScorecardForm.php

```diff
@@ -65,6 +65,8 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
+    // The block should always show the most actual data.
+    // We can't cache it.
```

Comment:
we have user cache tags, or user context. https://www.drupal.org/docs/8/api/cache-api/cache-contexts
Let's work on this after release.